### ISSUE TITLE
Documentation: please don't autoplay a video with sounds

### ISSUE THREAD

**Issue by nitro2k01** [ManimCommunity/manim#2879]
The documentation for `Scene` contains an example video called SoundExample, which as the name suggests is an example of using sounds in Manim. However, this video is set to autoplay and loop. This confused me at first since I didn't know what produced the sound. You have to scroll down a fair bit to see that there's an audio example. This is annoying if you just want to read that page in the documentation.

It doesn't always autoplay: In Firefox, it seems to never autoplay. In Chrome, it doesn't autoplay if you go directly to the page, but it does autoplay if you come from the [parent page](https://docs.manim.community/en/stable/reference/manim.scene.scene.html) and click Scene, for me.

The behavior might differ further between browsers. But the HTML code does request the video to be autoplayed, so it will be autoplayed sometimes in various browsers. Please make sure that this and any other example videos in the documentation that play sound don't autoplay.
```html
<video class="manim-video" controls loop autoplay src="./SoundExample-1.mp4"></video>
```

**Comment by behackl** [comment]
I think this is a reasonable request.

Currently, there is no way for finer control of whether individual videos autoplay or not: https://github.com/ManimCommunity/manim/blob/cd40bc3baf2dfa3e46d7f2817275282200fd464f/manim/utils/docbuild/manim_directive.py#L385

The way to go is to add a new option, `noautoplay` to the directive that modifies the template accordingly. The option should then be set for the `Scene.add_sound` example.

**Comment by nitro2k01** [comment]
Perhaps a simpler option that doesn't require special casing in the template is to add the `muted` attribute to the tag?

**Comment by OwenSamuel** [comment]
This issue has been fixed in #2911 but I do not know how to close it
